openassessment/templates/openassessmentblock/edit/oa_edit_staff_assessment.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+{% load tz %}
```

Comment:
Nit: tz not needed here (spaceless probably not either, but I'm OK with that one staying).